cards/src/getUsers.js

```diff
@@ -0,0 +1,19 @@
+import { UserCard } from "./UserCard";
```

Comment:
You can remove this import statement, since this function doesn't actually use the `UserCard`.